pkgs/development/python-modules/milksnake/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, python3Packages, buildPythonPackage, fetchurl }:
+
+buildPythonPackage rec {
+  pname = "milksnake";
+  version = "0.1.1";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
```

Comment:
fetchPypi

Reply:
Does not work. I tested it, but it can't find the package. That's why I used `fetchurl`.

Reply:
because you request a zip. It needs `extension = "zip";'